src/lib/y2network/config_reader/connection_config/sysconfig_handlers/wlan.rb

```diff
@@ -24,7 +24,7 @@ module ConfigReader
     module ConnectionConfig
       # This class is able to build a ConnectionConfig::Wireless object given a
       # SysconfigInterfaceFile object.
-      class SysconfigWireless
+      class Wlan
```

Comment:
missing sysconfig handler namespace?

Otherwise this change looks reasonable for me.

Reply:
btw I expect in future we will share some parts in mixin or common ancestor for things that is used for every kind of device.